src/peft/helpers.py

```diff
@@ -111,3 +120,92 @@ def update_signature(model: PeftModel, method: str = "all") -> None:
         update_generate_signature(model)
     else:
         raise ValueError(f"method {method} is not supported please choose one of ['forward', 'generate', 'all']")
+
+
+
+def create_loraplus_optimizer(model: PeftModel, optimizer_cls: type[Optimizer], optimizer_kwargs: dict) -> Optimizer:
+    """
+    Creates a LoraPlus optimizer.
+    Implementing LoRA+ https://arxiv.org/abs/2402.12354
+    Reference: https://github.com/nikhil-ghosh-berkeley/loraplus/
+
+    Args:
+        model (`torch.nn.Module`): The model to be optimized.
+        optimizer_cls (`torch.optim.Optimizer`): The optimizer class to be used.
+        optimizer_kwargs (`dict`): Additional keyword arguments to be passed to the optimizer.
+            - **loraplus_lr_ratio** (`float`): The ratio of the learning rate to be used for the embedding layer. Defaults to loraplus_lr_ratio
+            - loraplus_lr_embedding (`float`): The learning rate to be used for the embedding layer. Defaults to loraplus_lr_embedding
+    """
+    from .tuners.lora.layer import Embedding
+    loraplus_lr_ratio = optimizer_kwargs.pop("loraplus_lr_ratio")
+    loraplus_lr_embedding = optimizer_kwargs.pop("loraplus_lr_embedding")
```

Comment:
Instead of poppng `loraplus_lr_embedding` from the kwargs, make it an explicit argument of the function please, with default value `1e-6`.